.github/workflows/ci.yml

```diff
@@ -0,0 +1,83 @@
+name: Continuous Integration
+on:
+  push:
+    branches:
+      - main
```

Comment:
`development` branch here as well? Or have the Github settings been modified to enforce the PR is off of head?

Reply:
```
on:
  push:
    branches:
      - main
  pull_request:
```
means run on 
- push to `main` branch (in case of urgent bug fixes we might push directly without a PR
- on every opened PR

But I've added `development` still just in case someone pushes there directly as we cannot protect that branch atm.